website/docs/specification/DigitalIdentityAnchor.md

```diff
@@ -0,0 +1,242 @@
+| DIA-05 | Scope List| The DIA MUST include a list of scope URIs that unambiguously define the authorised role(s) of the member in the register so that verifiers can confirm the scope of the membership. | `scopeList` property |
+| DIA-06 | Register Type| The DIA MUST specify the register type so that verifiers can understand the context of the `scopeList`| `registerType` property|
+| DIA-07 | DIA Discovery| The DIA SHOULD be discoverable given either the DID or the registeredID| [DIA Discovery](#dia-discovery)|
+
+The examples below help to clarify the application of DIA-05 and DIA-06.
+
+## Logical Model
+
+The Digital Identity Anchor logical model.
+
+![Digital Product Passport data model](DigitalIdentityAnchor.svg)
+
+### Core Vocabulary Documentation
+
+The [UNTP core types vocabulary](https://jargon.sh/user/unece/untp-core/v/0.5.0/artefacts/readme/render) defines the uniquely identified Linked Data entities such as Product, Location, Facility, Party, Standard, Regulation, Criteria, Declaration, Attestation, Endorsement. The Digital Identity Anchor only re-uses the UNTP core `VerifiableCredential` and `Identifier` classes.
+
+
+### DIA Documentation
+
+The [DIA documentation](https://jargon.sh/user/unece/DigitalIdentityAnchor/v/0.1.0/artefacts/readme/render) provides a technology-neutral definition of classes, properties and code lists in the DIA model.
+
+## Implementation Guidance
+
+This section provides sample JSON-LD snippets for each DIA component.
+
+### Digital Identity Anchor
+
+The Digital Identity Anchors is a Verifiable credential. Please refer to [DPP VC Guidance](DigitalProductPassport.md#verifiable-credential) for information about the use of the verifiable credentials data model for UNTP.  THe issuing party for the VC MUST be the authority that owns or operates the identity register. 
+
+### Registered Identity
+
+The registered identity class represents the registry member. For example, in a national business register, the registered identity would be one of the registered businesses.
+
+* `type` MUST contain constant value array `["RegisteredIdentity", "Identifier"]` which declares to linked data processors that this is a registered identity which is a sub-type of identity.
+* the `id` MUST contain the globally unique member identifier as maintained in the register.  Often this will be a web link to the member entry.
+* `name` is the human readable registered entity name and SHOULD match the registered name as it appears in the authoritative register.
+* `registeredId` contains the simple identifier value that is unique within the register (but may not be globally unique) for example the VAT registration number.
+* `idScheme` identifies the authoritative register.  If the identity scheme is registered with UN/CEFACT then the `idScheme.id` MUST match the `identityRegister.id` in the UN/CEFACT scheme register.
```

Comment:
This feels inelegant.

Reply:
As mentioned above, I would rather see the organization's did as the `id` value.

Reply:
yes - will update the model and generated artefacts accordingly